nifi-marklogic-processors/pom.xml

```diff
@@ -49,6 +49,12 @@
       <groupId>com.marklogic</groupId>
       <artifactId>marklogic-data-hub</artifactId>
       <version>6.1.1</version>
+      <exclusions>
```

Comment:
I think this is too risky, even if our tests pass. The RestTemplate class comes from this package, and ml-app-deployer makes heavy use of it, and marklogic-data-hub depends on ml-app-deployer for talking to the Manage API. I don't think that happens when running a flow, but I don't know if that's always the case. There are also imports in the DataHubImpl class that start with `org.springframework.web`. 

I think we just need an exception for these - and we can probably reuse the DHF one for Spring 6 - https://progresssoftware.atlassian.net/wiki/spaces/ENGINEERING/pages/1259667487/Security+exception+for+Data+Hub+6.1.2+release . Can just reference that in the JIRA ticket for the NiFi 2.0.0 connector release.